tests/v2/08_full_workflow/common/common.py

```diff
@@ -17,49 +14,17 @@ def _task_name_to_id(task_name: str, task_list: list[dict[str, Any]]) -> int:
     return task_id
 
 
-PREFIX = "/api/v2"
-
-backends_available = list(_backends.keys())
-
-
-@pytest.mark.parametrize("backend", backends_available)
-async def test_full_workflow(
-    client,
+async def full_workflow(
     MockCurrentUser,
-    testdata_path,
-    tmp777_path,
+    client,
     project_factory_v2,
-    dataset_factory_v2,
     workflow_factory_v2,
-    backend,
-    override_settings_factory,
-    tmp_path_factory,
-    fractal_tasks_mock,
-    request,
+    dataset_factory_v2,
+    user_kwargs: dict = {},
```

Comment:
Let's avoid mutable default arguments.
The typical solution here is `Optional[dict] = None`, and then handle the `None` from the function code.